readthedocs_theme/templates/includes/topnav.html

```diff
@@ -67,6 +70,27 @@
   </a>
 {% endmacro %}
 
+{# Application links to sign up #}
+{% macro menu_sign_up() %}
+  <a class="item" data-analytics="community-signup" href="https://readthedocs.org/accounts/signup/">
+    <i class="fad fa-people-group primary icon"></i>
+    Read the Docs Community
+
+    <p class="ui mini grey text">
+      <code>https://readthedocs.org</code>
+    </p>
```

Comment:
The sign up modal was meant to cover this particular issue, and the modal does a much better job than just a domain name at describing the platform's purpose and target audience. For that, I still think all sign up buttons should trigger the sign up modal instead linking directly to our websites.

This gives us a chance to describe things thoroughly, and maybe the user doesn't need to remember the domain name at all after being introduced to our platforms more thoroughly.

Reply:
Agreed. Any reason we don't have a login modal? Because presumably users know where they want to go once they're logging in?

Reply:
It's definitely an option, and was something I thought about trying originally, but wanted to make sure the modal pattern worked well first. I could see it annoying users too.

I landed at wanting to add some sort of platform detection (or naive platform detection like just storing a cookie when the user clicks signup/login for either community/commercial). This would make the login button either go directly to the correct site, or could instead only show a modal when there is no cookie.

Definitely some improvements here as we go, I'm mostly waiting to see what the usage looks like at the moment.